runtime/gc_vlhgc/RegionBasedOverflowVLHGC.cpp

```diff
@@ -224,7 +224,7 @@ MM_RegionBasedOverflowVLHGC::overflowItemInternal(MM_EnvironmentBase *env, void
 				 * To avoid adding constructing object 
 				 */
 				envVLHGC->getGCEnvironment()->_ownableSynchronizerObjectBuffer->add(envVLHGC, objectPtr);
-				if (envVLHGC->_cycleState->_shouldRunCopyForward) {
+				if (MM_CycleState::CT_PARTIAL_GARBAGE_COLLECTION == envVLHGC->_cycleState->_collectionType) {
```

Comment:
I remember discussing this long ago... So we agreed even if real hybrid mode was performed (there was real regions that were Marked) we will gather all relevant stats under _copyForwardStats? I think I'm ok with it, I just want to make sure that's we agreed on and we stick with the decision everywhere (soft ref stats etc). We should probably state it somewhare in the code as well, as a comment (in declaration of _copyForwardStats in EnvVLHGC).